raptorWeb/authprofiles/views.py

```diff
@@ -0,0 +1,347 @@
+from django.shortcuts import render, redirect
+from django.views.generic import TemplateView
+from django.contrib import messages
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth import authenticate, login, logout
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.http import HttpResponseRedirect, HttpResponse
+from django.core.exceptions import ObjectDoesNotExist
+from os.path import join
+from logging import getLogger
+
+from raptorWeb import settings
+from authprofiles.forms import UserForm, UserProfileInfoForm, UserLoginForm, DiscordUserInfoForm
+from authprofiles.models import User, UserProfileInfo, DiscordUserInfo
+from raptormc.util import viewContext
+from authprofiles.util import discordAuth
+from raptormc.jobs import player_poller
```

Comment:
The authprofiles app should not be coupled to the `raptormc` app or `jobs.py` at all. 

Any information about profiles (list of users and their profile picture URLs currently) that is put into template context by jobs.py should be moved directly to this app.